### DISCUSSION TITLE
Slight difference in results when using -corners

### DISCUSSION THREAD

**Discussion by kareefardi** [The-OpenROAD-Project/OpenROAD#6174] [Q&A]
[test.tar.gz](https://github.com/user-attachments/files/17791107/test.tar.gz)
This is a test case of two scripts running global placement. Both functionally should be identical. The difference is one uses `-corner` for `set_wire_rc` while the other doesn't.

For the case using `-corner`, `set_layer_rc` is needed. The code attempts to fetch the default values in the tlef and reuse them in `set_layer_rc` after doing the necessary unit conversions. 

There seems to be a slight difference in the output generated from both runs (notice the reported slacks and HPWL) and I am wondering if this a round-off error, a methodology error at my end or something else related to OR itself.

**Comment by maliberty** [comment]
Converted to issue #6175
